book/testing.md

```diff
@@ -1,8 +1,61 @@
-# Testing in Nushell (preview)
+# Testing in Nushell
```

Comment:
This is not really "testing in nushell" is it? It looks like it's testing in nushell's standard library, or testing in the nushell language.

So, for me, the filename is misleading and this topic is misleading.

Reply:
This chapter is how to test your code in Nushell. (I mean, in production environment.) A documentation of the unit testing framework available in Nushell. If you write a code in Nushell, how to test it in Nushell.

Can you suggest a better filename and topic for it? I would be happy to rename it to whatever you want.

Reply:
Maybe, "Testing your nushell code"? I'm using "code" to help cover scripts, modules, overlays (at some point), etc.

Reply:
I renamed the topic as you suggested. 👍

Reply:
Thanks